expense-rescript/src/components/IncomeExpenseForm/IncomeExpenseForm.css

```diff
@@ -0,0 +1,38 @@
+.income-expense-form {
+  box-shadow: var(--box-shadow);
```

Comment:
Check if this providing any additional value?

Reply:
No, it's not providing any additional value.